content/influxdb3/cloud-dedicated/admin/users/admin-ui.md

```diff
@@ -28,13 +28,18 @@ Use the {{% product-name %}} Admin UI to manage users and control access to your
 
 ## Access the Users page
 
-1. Access the {{% product-name %}} Admin UI at [console.influxdata.com](https://console.influxdata.com).
-   If you don't have login credentials, [contact InfluxData support](https://support.influxdata.com).
-2. Log in using the credentials provided by InfluxData.
```

Comment:
I proposed condensing this throughout as I thought it might be too wordy; I'll revert if we think the specific instructions on selecting the account/cluster are needed on each page